manifests/q/quick-lint/quick-lint-js/2.4.1/quick-lint.quick-lint-js.installer.yaml

```
@@ -0,0 +1,22 @@
+# To the extent possible under law, Matthew "strager" Glazar
+# has waived all copyright and related or neighboring rights
+# to _quick-lint-js winget manifests_. This work is
+# published from United States.
+# http://creativecommons.org/publicdomain/zero/1.0/
+
+PackageIdentifier: quick-lint.quick-lint-js
+PackageVersion: 2.4.1
```

```suggestion
PackageVersion: 2.4.1.0
```
This must match Control Panel.